addons/account/security/account_security.xml

```diff
@@ -295,6 +295,7 @@
         <field name="model_id" ref="model_account_invoice_send"/>
         <field name="domain_force">[(1, '=', 1)]</field>
         <field name="groups" eval="[(4, ref('account.group_account_invoice'))]"/>
+        <field name="perm_unlink" eval="False"/>
```

Comment:
I don't understand how these changes were crafted.

Unlinking rights on a transient model? Why would we need to change that?